src/basic_commands/mapping_files.md

```diff
@@ -38,32 +44,34 @@ Usage: o[?]   # Open files and handle opened files
 Prepare a simple layout:
 
 ```bash
-$ rz-bin -l /bin/ls
-[Linked libraries]
-libselinux.so.1
-librt.so.1
-libacl.so.1
+$ rz-bin -l hello_world
+[Libs]
+library   
+----------
 libc.so.6
-
-4 libraries
 ```
 
 Map a file:
 
 ```
-[0x00001190]> o /bin/zsh 0x499999
+[0x00001100]> o /bin/sh 0x499999
 ```
 
 List mapped files:
 
 ```
 [0x00000000]> ol
-- 6 /bin/ls @ 0x0 ; r
-- 10 /lib/ld-linux.so.2 @ 0x100000000 ; r
-- 14 /bin/zsh @ 0x499999 ; r
+ 3 - r-x 0x00003d48 /home/user/playground/book/examples/hello_world/hello_world
+ 4 - r-x 0x00000070 vfile://0/reloc-targets
+ 5 - rw- 0x00000008 null://8
+ 6 - r-x 0x00003d48 vfile://0/patched
+ 7 * r-x 0x000d5b68 /bin/sh
+ 8 - r-- 0x00000bf0 vfile://1/reloc-targets
```

Comment:
I think it probably worth to explain what `vfile://` and `null://` are. `vfile` is a virtual file, that is often automatically created to patch relocations and could also be created manually, if needed. It was created to avoid modifying the original file/IO ranges.